Tasks/VSMobileCenterUpload/task.json

```diff
@@ -0,0 +1,149 @@
+{
+    "id": "B832BEC5-8C27-4FEF-9FB8-6BEC8524AD8A",
+    "name": "VSMobileCenterUpload",
+    "friendlyName": "Visual Studio Mobile Center Upload",
+    "description": "Upload mobile app packages to Visual Stuidio Mobile Center",
+    "helpMarkDown": "Upload mobile app packages to Visual Stuidio Mobile Center",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "groups": [
+        {
+            "name": "symbols",
+            "displayName": "Symbols",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "serverEndpoint",
+            "type": "connectedService:vsmobilecenter",
+            "label": "Mobile Center Connection",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select the service endpoint for your Visual Studio Mobile Center connection. To create one, click the Manage link and create a new Service Endpoint."
+        },
+        {
+            "name": "appSlug",
+            "type": "string",
+            "label": "App Slug",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The app slug is in the format of {username}/{app_identifier}.  To locate {username} and {app_identifier} for an app, click on its name from https://mobile.azure.com/apps, and the resulting url is in the format of https://mobile.azure.com/users/{username}/apps/{app_identifier}"
+        },
+        {
+            "name": "app",
+            "type": "filePath",
+            "label": "Binary File Path",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Relative path from the repo root to the apk or ipa file you want to publish"
+        },
+        {
+            "name": "symbolsType",
+            "type": "pickList",
+            "label": "Symbols Type",
+            "required": false,
+            "defaultValue": "Apple",
+            "groupName": "symbols",
+            "options": {
+                "Apple": "Apple",
+                "AndroidJava": "Android (Java)",
+                "AndroidNative": "Android (Native)",
+                "Windows": "Windows"
```

Comment:
We should change "Android (Native)" to "Android (native C/C++)" since so many people have questioned this.